### PR TITLE
Site V2

### PR DESCRIPTION
**Pull request by jaspermayone**

### CODE REVIEW

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+import "@/styles/globals.css";
 
 export default function App({ Component, pageProps }: AppProps) {
   return (
     <>
-    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+      <Head>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+      </Head>
       <Component {...pageProps} />
```
Comment (aramshiva) [inline src/pages/_app.tsx:12]:
```suggestion
      <AnimatePresence>
      <Component {...pageProps} />
      </AnimatePresence>
```

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -1,15 +1,15 @@
-import "@/styles/globals.css";
 import type { AppProps } from "next/app";
-import Script from "next/script";
+import Head from "next/head";
```
Comment (aramshiva) [inline src/pages/_app.tsx:2]:
```suggestion
import Head from "next/head";
import AnimatePresence from "framer-motion";
```

**Review by aramshiva — CHANGES_REQUESTED** [review]

**Comment by aramshiva** [comment]
What’s the error in logs?

**Comment by jaspermayone** [comment]
@aramshiva 
CleanShot 2024-09-15 at 09 04 34@2x
CleanShot 2024-09-15 at 09 05 15@2x

**Comment by jaspermayone** [comment]
CleanShot 2024-09-15 at 09 05 45@2x
its complaining about animate presence in the app.ts

**Comment by jaspermayone** [comment]
I fixed it, but am still getting keyframe errors.

**Comment by jaspermayone** [comment]
World's most insane PR right here :)

**Comment by jaspermayone** [comment]
@aramshiva I made a new "gallery" type view on /friends for something I am working on, but the files won't be numbered like the photo gallery on my photo page for the v1 site. I am storing the images in a folder, is there any way we can adapt the api to work for that?

**Comment by aramshiva** [comment]
i dont think it needs to be numbered

**Comment by jaspermayone** [comment]
Is there a way to loop through the files in a folder or should I do them individually?

**Comment by jaspermayone** [comment]
@aramshiva can you look at [88e0a4e](https://github.com/jaspermayone/website/pull/26/commits/88e0a4e59dbbd8506bb981dfc3895f91e9078c8c) please? The /friends page seems to be loading in images in a weird way... it might just be the number of images, but one side seems to lack images down towards the bottom.

**Comment by aramshiva** [comment]
> @aramshiva can you look at [88e0a4e](https://github.com/jaspermayone/website/pull/26/commits/88e0a4e59dbbd8506bb981dfc3895f91e9078c8c) please? The /friends page seems to be loading in images in a weird way... it might just be the number of images, but one side seems to lack images down towards the bottom.

You can have the Image with a [Suspense](https://nextjs.org/docs/app/building-your-application/routing/loading-ui-and-streaming) or change the [Priority](https://nextjs.org/docs/pages/building-your-application/optimizing/images#priority)

**Comment by jaspermayone** [comment]
Perfect! Ty!

**Comment by jaspermayone** [comment]
MERGE TIMEEEE

**Pull request merged**